sound/soc/sof/topology.c

```diff
@@ -2853,6 +2853,10 @@ static int sof_link_load(struct snd_soc_component *scomp, int index,
 	if (!link->no_pcm) {
 		link->nonatomic = true;
 
+		/* set trigger order */
+		link->trigger[0] = SND_SOC_DPCM_TRIGGER_POST;
```

Comment:
Shouldn't this be set in machin drivers, or at least in Intel DSP specific code (as this is a Intel DSP limitation)? I see most drivers setting this in machine drivers.

Reply:
@kv2019i we SOF does not define the FE DAI links in the machine driver. They are created when the pcm elems in topology are parsed.